### DISCUSSION TITLE
set operators : CORRESPONDING BY

### DISCUSSION THREAD

**Discussion by jimdickson007** [duckdb/duckdb#14068] [Ideas]
This is actually part of SQL standards - described here informally 

https://blog.jooq.org/a-rarely-seen-but-useful-sql-feature-corresponding/

**Comment by Tishj** [comment]
404 not found on your link

**Reply by jimdickson007** [reply]
Thanks - updated link in original post
Plus another link (partially visible text of standards book)

https://books.google.co.th/books?redir_esc=y&id=ZOOMSTZ4T_QC&q=%22corresponding+by%22#v=snippet&q=%22corresponding%20by%22&f=false
